### PR TITLE
Remove BrowserPanel and SplitPanel from app

### PR DESCRIPTION
**Pull request by Skaty**

### CODE REVIEW

File: src/main/java/seedu/task/ui/MainWindow.java [hunk 1]
```diff
@@ -42,9 +41,9 @@
     private Scene scene;
 
     private String taskManagerName;
-
+    
     @FXML
-    private AnchorPane browserPlaceholder;
+    private VBox taskList;
```
Comment (advaypal) [inline src/main/java/seedu/task/ui/MainWindow.java]: taskList? What is this for

**Review by advaypal — COMMENTED** [review]
Made comments

File: src/main/java/seedu/task/ui/MainWindow.java (hunk 1 above)
Comment (Skaty) [reply src/main/java/seedu/task/ui/MainWindow.java]: It's defined in the FXML. I'll see if it works even after removing it.

**Comment by advaypal** [comment]
Looks good to me. But why the coverage decrease?

**Comment by Skaty** [comment]
@advaypal: The BrowserPanel class is still inside the codebase, just unused.

**Review by advaypal — APPROVED** [review]

**Comment by Skaty** [comment]
Tests not successful because of date dependent tests.

**Pull request merged**